src/Network/Network/Generated/Models/PSLoadBalancingRule.cs

```diff
@@ -64,6 +65,8 @@ public partial class PSLoadBalancingRule : PSChildResource
         [JsonProperty(Order = 1)]
         public PSResourceId BackendAddressPool { get; set; }
         [JsonProperty(Order = 1)]
+        public List<PSResourceId> BackendAddressPools { get; set; }
```

Comment:
Why here keep BackendAddressPool and BackendAddressPools at the same time? I think BackendAddressPools is enough

Reply:
this is just how our backend supports it. when it gets passed to nrp, they are actually different how they are treated.

Reply:
Is there any workaround that can avoid providing BackendAddressPool and BackendAddressPools at the same time?